opendcs-web-client/src/main/webapp/resources/js/datatables/helpers.js

```diff
@@ -103,11 +103,9 @@ function createActionDropdown(applicableActions)
 function setPropSpecsMeta()
 {
     //seasonMeta
-    var params = { 
-            "opendcs_api_call": "seasons"
-    };
+    var params = {};
     $.ajax({
-        url: `../api/gateway`,
+        url: `${window.API_URL}/seasons`,
```

Comment:
We may want to wrap these within a larger API. I know the cookies will get get sent, but I think the authorization header may not always be sent.